tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,18 @@
+// This file is used as part of the bootstrap-sass-loader customization
```

Comment:
This file needs to be renamed. It's not bootstrap specific.

Should be called.

`_rails-sass-customization.scss`